Xamarin.Forms.Core/BindableObject.cs

```diff
@@ -80,6 +93,11 @@ void ClearValue(BindableProperty property, bool fromStyle, bool checkAccess)
 			}
 		}
 
+		protected BindableObject()
+		{
+			DispatcherManager.Current.Init();
```

Comment:
Getting `Current` will call `Init()` anyway, so I don't think it needs to be called here.

Reply:
I'm looking at all the comments, however in this case we need to call the init at the moment bindable is created so that Current has the correct thread value and be guaranteed to have value when called within a Task.Run, for example.